src/Uno.UI/UI/Xaml/Controls/WebView/Core/CoreWebView2.iOSmacOS.cs

```diff
@@ -0,0 +1,39 @@
+﻿#nullable enable
+
+using System;
+using System.Linq;
+using Uno.UI.Xaml.Controls;
+using Uno.Foundation.Logging;
+using Windows.UI.Xaml.Controls;
+
+#if __IOS__
+using UIKit;
+using _View = UIKit.UIView;
+#else
+using AppKit;
+using _View = AppKit.NSView;
+#endif
+
+namespace Microsoft.Web.WebView2.Core;
+
+public partial class CoreWebView2
+{
+	internal INativeWebView? GetNativeWebViewFromTemplate()
+	{
+		var nativeWebView = (_owner as _View)
+			.FindSubviewsOfType<INativeWebView>()
+			.FirstOrDefault();
```

Comment:
Is `_owner` guaranteed to be non-null of type `UIView`/`NSView`?

If yes: then consider a cast instead of `as`.
If no: then a null check and a test are missing.